app/src/main/res/values/styles.xml

```diff
@@ -13,6 +13,19 @@
         <item name="windowNoTitle">true</item>
     </style>
 
+    <style name="One_line">
+        <item name="android:ellipsize">end</item>
+        <item name="android:maxLines">1</item>
+        <item name="android:singleLine">true</item>
+    </style>
+
+    <style name="Default_margins">
+        <item name="android:layout_marginLeft">10dp</item>
```

Comment:
`left` and `right` don't take into account RTL (right-to-left) locales, so it's better to use `start` and `end`.